src/Runner.Listener/JobDispatcher.cs

```diff
@@ -512,14 +512,14 @@ private async Task RunAsync(Pipelines.AgentJobRequestMessage message, string orc
                             var completedTask = await Task.WhenAny(renewJobRequest, workerProcessTask, Task.Delay(-1, jobRequestCancellationToken));
                             if (completedTask == workerProcessTask)
                             {
+                                var detailInfo = string.Join(Environment.NewLine, workerOutput);
+                                Trace.Info(detailInfo);
```

Comment:
This will dump the entire collection of Worker logs in a single string - it can easily be thousands of lines long.
Should we consider trying to print worker logs asap?

Reply:
In case the runner loses connection / its pod is destroyed, waiting until the end of the job risks losing the logs alltogether